api_tests/registrations/views/test_registration_files_list.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.json_api_test_app import JSONAPITestApp
+from api.base.settings.defaults import API_BASE
+from api_tests import utils as api_utils
+from tests.base import ApiTestCase
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    RegistrationFactory
+)
+
+@pytest.mark.django_db
+class TestRegistrationFilesList(object):
+
+    @pytest.fixture(autouse=True)
+    def setUp(self):
```

Comment:
Drive-by: you can use `setup_method` here: https://docs.pytest.org/en/latest/xunit_setup.html#method-and-function-level-setup-teardown